code/modules/busy_space/systems.dm

```diff
@@ -0,0 +1,603 @@
+
+//human independents
+/datum/lore/system/natuna
+	name = "Natuna Barisal" //we have unicode support but i think radio might still shit itself
+	desc = "Seceeded from SolGov over all the racism and casteism. Now it's home to pirates and also got invaded by the Hegemony"
+	planets = list("Natuna Barisal c") //its this or 'the planet' or 'natuna barisal, natuna barisal'-- the planet and the system share a name
+	locations = list(
+		new /datum/lore/location("Dhana in Natuna Barisal", list(ATC_TYPICAL)),
+		new /datum/lore/location("a human settlement on Natuna Barisal c", list(ATC_TYPICAL, ATC_DIPLO)),
+		new /datum/lore/location("an Ue-Katish settlement on Natuna Barisal c", list(ATC_TYPICAL, ATC_DIPLO)),
+		new /datum/lore/location("an abandoned Hegemony outpost on Natuna Barisal c", list(ATC_SALVAGE, ATC_SCI)),
+		new /datum/lore/location("a Ue-Katish trade ship in Natuna Barisal", list(ATC_TYPICAL, ATC_DIPLO)), //"trade ship"
+		new /datum/lore/location("an orbital dockyard in Natuna Barisal", list(ATC_TYPICAL)), // "dockyard"
+		new /datum/lore/location("a survey site on Natuna Barisal c", list(ATC_SCI)),
+		)
+/datum/lore/system/natuna/New()
+	planetary_destinations -= list("a colony", "a city")
+	planetary_destinations += "a settlement" //this is the language used on the natuna page and i think that makes for fun regional flavor
+	..()
+
+
+/datum/lore/system/phact
+	name = "Phact"
+	desc = "They built a replica of Edo Castle in a dome. That's really all there is to say on the matter."
+	planets = list("New Kyoto", "Hokkaido", "Ryukyu")
+	locations = list(
+		new /datum/lore/location("the Imperial Palace in New Kyoto, Phact", list(ATC_LUX, ATC_DIPLO)),
+		)
+
+/datum/lore/system/casinis_reach
+	name = "Casini's Reach"
+	desc = "Anarchists who have managed to frame-perfect parry every terrible thing to have happened to humanity in the last 100 years."
+	planets = list("Santiago, Pontes subsystem", "Cotopaxi, Pontes subsystem") //theyre not planets but i think this will work fine
+	locations = list(
+		new /datum/lore/location("Rey Pedro, Pontes subsystem, Casini's Reach", list(ATC_TYPICAL)),
+		new /datum/lore/location("Santa Maria, Pontes subystem, Casini's Reach", list(ATC_TYPICAL)),
+		new /datum/lore/location("Aconcagua, Pontes subsystem, Casini's Reach", list(ATC_TYPICAL)),
+		new /datum/lore/location("Campinas, Pontes subsystem, Casini's Reach", list(ATC_TYPICAL)),
+		new /datum/lore/location("a commune in the Diego Belt, Casini's Reach", list(ATC_TYPICAL)),
+		new /datum/lore/location("a large commune on Santiago, Pontes subsystem, Casini's Reach", list(ATC_ALL_CIV, ATC_DIPLO)), //kinda kludgy but that lets ppl run diplomatic missions here
+		new /datum/lore/location("an abandoned mineshaft on Santa Maria, Pontes subsystem, Casini's Reach", list(ATC_MED, ATC_SALVAGE)),
+		new /datum/lore/location("the Unbound colony on Santa Maria, Pontes subsystem, Casini's Reach", list(ATC_TYPICAL)),
```

Comment:
```suggestion
		new /datum/lore/location("the Unbound colony on Santa Maria, Pontes subsystem, Casini's Reach", list(ATC_TYPICAL))
```
Have I made my point with the commas yet :p

Reply:
you could have just said to fix trailing commas I could have done all this :P